templates_ark/example_pixel_clustering.ipynb

```diff
@@ -250,6 +250,7 @@
     "    segmentation_dir,\n",
```

Comment:
<p>pixel_cluster_prefix should get added here as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/616/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>